src/connectors/toString.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Return a string representing the given connector.
+ *
+ * @param {connector} connector - the connector of reference
+ * @returns {string} string representation
+ */
+const toString = (connector) => {
+  const point = connector.point
+  const axis = connector.axis
+  const normal = connector.normal
+  return `connector: point: [${point[0].toFixed(7)}, ${point[1].toFixed(7)}, ${point[2].toFixed(7)}],  axis: [${axis[0].toFixed(7)}, ${axis[1].toFixed(7)}, ${axis[2].toFixed(7)}], normal: [${normal[0].toFixed(7)}, ${normal[1].toFixed(7)}, ${normal[2].toFixed(7)}]`
```

Comment:
yay for es6 strings , well done !
hmm just a nitpick, but could you please format it across multiple lines for readability ? (or it might be the github review page messing it up, in which case, please disregard)